src/scenes/ProgressReports/EditForm/Steps/Media/MediaInfoForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, TextField } from '@mui/material';
+import {
+  ProgressReportMediaCategoryLabels,
+  ProgressReportMediaCategoryList,
+} from '~/api/schema/enumLists';
+import { labelFrom } from '~/common';
+import { Form, SelectField } from '~/components/form';
+import { minLength, required } from '~/components/form/validators';
+import { useCallback, useState } from 'react';
+import { ProgressReportMediaCategory } from '~/api/schema.graphql';
+
+// TODO type submitForm
+// @ts-ignore
+export const MediaInfoForm = ({
+  submitForm,
+  disabled,
+  initialValues,
+}: {
+  submitForm: any;
+  disabled: boolean;
+  initialValues: {
+    category: ProgressReportMediaCategory | null;
+    caption: string | null;
+  };
+}) => {
+  const [category, setCategory] = useState(initialValues.category);
+  const [caption, setCaption] = useState(initialValues.caption);
+  const updateMedia = useCallback(() => {
+    submitForm({ variables: {} });
+  }, []);
+  return (
+    <Form onSubmit={submitForm}>
+      <Box sx={{ p: 1, m: 1, flexGrow: 2 }}>
```

Comment:
`<Form>` is just the JS implementation, we have to render the html form tag ourselves. This is good because it gives us styling flexibility.
```suggestion
    <Form onSubmit={submitForm}>
      {({ handleSubmit }) => (
        <Box component="form" onSubmit="handleSubmit" ... />
      )}
```
I would also not do the padding & margin like you have. Instead have that outer `px` on the parent do 4 like the PromptResponseAccordions have. And then use `gap` here for internal spacing.